blog/2020-09-21-setup-bitcoin-cloud-node.md

```diff
@@ -0,0 +1,302 @@
+If you want to go for a cheaper option you can sync on e.g. an external disk at home or on a Raspy.
+
+Instead of doing the setup yourself you can opt for using a pre-configured, pre-synced image provided on [Google Cloud Marketplace](https://console.cloud.google.com/marketplace/details/techlatest-public/bitcoin-fullnode).
+We opt for doing the setup ourselves to be in full control of what is going on.
+
+### Initial Google Instance Setup
+
+We refer to Bitcoin Core as bitcoind, as in bitcoin daemon, throughout this tutorial.
+
+#### Instance setup
+
+1. Create a `e2-standard-4 (4 vCPUs, 16 GB memory)` instance, provision it with `Debian`.
+2. [Add a 500 GB harddisk to the instance](https://cloud.google.com/compute/docs/disks/add-persistent-disk#create_disk) (you can do this during instance setup at the bottom or later).
+3. [Configure a static external IP address for the instance](https://cloud.google.com/compute/docs/ip-addresses/reserve-static-external-ip-address#assign_new_instance).
+
+#### Configure the Google Cloud firewall
+
+ℹ️ This tutorial assumes that you want to expose the RPC interface to the outside world, if you don't want to do this you can skip this step.
+
+In this step we expose the ports 8333 (and 8332) in the Google Cloud firewall.
+
+You will have to configure a `New Firewall` within the Network -> Firewall settings of the Google Comupte Engine. 
+
+1. In your instance detail view you should see `Network interfaces`, click on the `default` link under `Network`.
+2. Chose `Firewall` in the left hand menu
+3. Click `Create Firewall` on the top
+4. Define whatever name
+5. Change `Targets` to `All instances in the network` (or see Google Cloud docu)
+6. Set `Source IP ranges` to `0.0.0.0/0` (or see Google Cloud docu)
+7. Specify port under `tsc` to `8332-8333`
+8. Click `Create`
+
+The rule will automatically be applied to your instance (due to 5.).
+
+Note: Setting up an [`ingress`](https://cloud.google.com/vpc/docs/firewalls#direction_of_the_rule) (incoming from source to target) rule is what you need.
+
+### Getting bitcoind installed
+
+In this tutorial we are building bitcoind from source.
+This can be changed to e.g. download the binary or use [script](https://bitnodes.io/install-full-node.sh) like the one provided by [bitnodes.io](https://bitnodes.io/#join-the-network).
+
+This tutorial is based on the latest [UNIX build instructions](https://github.com/bitcoin/bitcoin/blob/master/doc/build-unix.md) for `v0.20.1` of the docs in the bitcoin repo.
+
+#### Prerequisites:
+
+Make sure `git` is installed:
+
+```shell script
+sudo apt-get install git
+```
+
+#### Install Dependencies
+
+First run:
+```shell script
+sudo apt-get install build-essential libtool autotools-dev automake pkg-config bsdmainutils python3
+```
+
+Then run:
+```shell script
+sudo apt-get install libevent-dev libboost-system-dev libboost-filesystem-dev libboost-test-dev libboost-thread-dev
```

Comment:
Why separating it?

Reply:
The official docu split this up because the second step is only necessary in some cases. Since it is necessary for us I combined this now.